app/javascript/src/Dashboard/tabs-widget.jsx

```diff
@@ -1,17 +1,36 @@
-// @flow
+const TAB_PRODUCTS = 'tab-products'
+const TAB_BACKENDS = 'tab-backends'
+
+function updateActiveTab (activeTab, inactiveTab) {
```

Comment:
I believe this widget exists because the dashboard tabs are rendered by rails and we are just adding the interactive functionality... it's a bit pitty tho..
Also, the code is very imperative... maybe we could enhance it managing a list and toggling the active class and defining the `active` one at initialization or by rails. 
I know PF needs all those classes and the tree rendered properly, but that doesn't mean we just can't make it simpler and the `addEventListener` part can be simplified for sure since its duplicated code.
Give it a thought and assess whether its possible to do it on time or not.

Reply:
@didierofrivia AFAIK it needs to be merged ASAP.
I'd rather create a new task to improve it in the near future.